pyhap/accessory.py

```diff
@@ -434,11 +467,11 @@ def __init__(self, display_name, mac=None, pincode=None,
                                      setup_id=setup_id)
         self.accessories = {}  # aid: acc
 
-    def set_sentinel(self, run_sentinel):
+    def set_sentinel(self, run_sentinel, aio_stop_event, event_loop):
         """Set the same sentinel to all contained accessories."""
-        super(Bridge, self).set_sentinel(run_sentinel)
+        super(Bridge, self).set_sentinel(run_sentinel, aio_stop_event, event_loop)
```

Comment:
No need for `super(Bridge, self).set_sentinel()`. `super().set_sentinel` is all that is needed.

Reply:
Addressed.